faq.py

```diff
@@ -0,0 +1,18 @@
+import json
+import re
+
+from telethon import events
+
+# TODO Make this better
+template = "The cause of this error is most likely {cause}. To fix it you need to {solution}"
+with open("faq.json", "r", encoding="utf-8") as out:
```

Comment:
Shouldn't that be `in`? Or, since that's reserved, `fd` for `file descriptor`, but definitely not `out`.